packages/material-ui/src/Chip/Chip.d.ts

```diff
@@ -45,8 +45,8 @@ export interface ChipTypeMap<P = {}, D extends React.ElementType = 'div'> {
       deletableColorSecondary?: string;
       /** Styles applied to the root element if `variant="outlined"`. */
       outlined?: string;
-      /** Styles applied to the root element if `variant="default"`. */
-      default?: string;
+      /** Styles applied to the root element if `variant="standard"`. */
+      standard?: string;
```

Comment:
This rename is not a breaking change, as it was added with https://github.com/mui-org/material-ui/pull/22161